Clarify PR Template settings behavior by git provider (#2743)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/git/pr-template.md

title: "PR template"
id: "pr-template"
---
## Configure pull request (PR) template URLs

When changes are committed on a branch in the IDE, dbt Cloud can prompt users to
open a new Pull Request for the code changes. To enable this functionality, ensure
that a PR Template URL is configured in the Repository details page in your
Account Settings. If this setting is blank, the IDE will prompt users to merge the changes directly into their default branch.

<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/configure-template-url-new-ide.png" title="Open a PR in the IDE"/>

### PR Template URL by git provider

The PR Template URL setting will be automatically set for most repositories, depending on the connection method.

> - If you are connecting to your repository using in-app integrations with your git provider or using the `Git Clone` method via SSH, this URL setting will be automatically populated and can be edited.

Reviewer: suggesting active voice 

```suggestion
- If you connect to your repository via in-app integrations with your git provider or the "Git Clone" method via SSH, this URL setting will be auto-populated and editable.
```

>  - If you are using a dbt Cloud Managed repository, this URL will not be set, and IDE will prompt users to merge the changes directly into their default branch.

Reviewer: ```suggestion
 - If you connect via a dbt Cloud [Managed repository](/docs/collaborate/git/managed-repository), this URL will not be set, and the IDE will prompt users to merge the changes directly into their default branch.
```
